app/Event/EventControllerProvider.php

```diff
@@ -81,6 +82,7 @@ function (Application $app) {
         $controllers->get('/{cdbid}/calsum', 'event_controller:getCalendarSummary');
 
         $controllers->patch('/{eventId}/subEvents', UpdateSubEventsStatusRequestHandler::class . ':handle');
+        $controllers->put('/{cdbid}/status', UpdateStatusRequestHandler::class . ':handle');
```

Comment:
(Same as https://github.com/cultuurnet/udb3-silex/pull/537#discussion_r551312650)

Reply:
I think this is also broken because you use `$eventId` in the request handler but `{cdbid}` here. In Silex the arguments need to have the same name. https://github.com/cultuurnet/udb3-silex/pull/537/files#diff-e1cc22686a32f871adef2a7a06a16fe90f4918ea13069346b390465d951e9e0aR30

Reply:
Yes, good spot!